charts/prl-ccd-definitions/values.preview.template.yaml

```diff
@@ -333,6 +381,22 @@ prl-cos:
       CORE_CASE_DATA_API_URL: http://${SERVICE_NAME}-ccd-data-store-api
       CCD_CASE_DOCS_AM_API: http://${SERVICE_NAME}-cdam
       ACA_SERVICE_API_BASEURL: http://${SERVICE_NAME}-aac-manage-case-assignment
+      AM_ROLE_ASSIGNMENT_API_URL: ${ROLE_ASSIGNMENT_URL}
       BUNDLE_URL: http://${SERVICE_NAME}-em-ccdorc
-      PRL_S2S_AUTHORISED_SERVICES: api_gw,prl_citizen_frontend,fis_hmc_api,ccd_data,prl_cos_api
+      PRL_S2S_AUTHORISED_SERVICES: api_gw,prl_citizen_frontend,fis_hmc_api,ccd_data,prl_cos_api,payment_app
       APP_ENV: "preview"
+
+postgresql:
+  enabled: true
+  flexibleserver: private-law-preview
+  setup:
+    databases:
+      - name: "pr-${CHANGE_ID}-data-store"
```

Comment:
I think we also need the following dbs
      - name: "pr-${CHANGE_ID}-draftstore"
      - name: "pr-${CHANGE_ID}-evidence"
      - name: "pr-${CHANGE_ID}-annotation"

Reply:
We dont have DM store setup on the preview its pointing to AAT - http://dm-store-aat.service.core-compute-aat.internal

Reply:
I meant draft-store

Reply:
Which service will be consuming it?